patches/net/minecraft/FileUtil.java.patch

```diff
@@ -27,7 +27,7 @@
 @@ -74,7 +_,7 @@
  
      public static boolean isPathPortable(Path p_133735_) {
-         for(Path path : p_133735_) {
+         for (Path path : p_133735_) {
```

Comment:
What happened here?

Reply:
Setup and unpack on a fresh pull yourself. It just happens. My guess is for some reason the patch file has it without the space that is actually there so every time someone unpacks it adds this file. I included it as part of this so that people don’t have to keep skipping including the patch when making PRs